doxia-core/src/test/java/org/apache/maven/doxia/sink/impl/SinkEventTestingSink.java

```diff
@@ -803,11 +488,6 @@ public void division(SinkEventAttributes attributes) {
         addEvent("division", new Object[] {attributes});
     }
 
-    @Override
-    public void verbatim() {
-        addEvent("verbatim");
-    }
-
     @Override
     public void verbatim(SinkEventAttributes attributes) {
         addEvent("verbatim", new Object[] {attributes});
```

Comment:
These changes I do not fully understand. Now the events are gone?

Reply:
No, the events are only added through the methods with the SinkEventAttributes arguments. The other ones just internally delegate to those (via the AbstractSink).